### ISSUE TITLE
memory error running openpose demo

### ISSUE THREAD

**Issue by rnunziata** [CMU-Perceptual-Computing-Lab/openpose#223]
I am on ubunt16, cuda8 cudnn6 , opencv3..get the same error with or without running as sudo

```
*** Error in `./build/examples/openpose/openpose.bin': free(): invalid next size (fast): 0x00007f3c0c000ba0 ***
======= Backtrace: =========
/lib/x86_64-linux-gnu/libc.so.6(+0x777e5)[0x7f3c61e547e5]
/lib/x86_64-linux-gnu/libc.so.6(+0x8037a)[0x7f3c61e5d37a]
/lib/x86_64-linux-gnu/libc.so.6(+0x82d52)[0x7f3c61e5fd52]
/lib/x86_64-linux-gnu/libc.so.6(posix_memalign+0x11d)[0x7f3c61e6471d]
/usr/local/lib/libopencv_core.so.3.3(_ZN2cv10fastMallocEm+0x2b)[0x7f3c5bf057ab]
/usr/local/lib/libopencv_core.so.3.3(_ZN2cv6String8allocateEm+0x1d)[0x7f3c5bd91a4d]
/home/rjn/opencv3-p3-code/people_pose/openpose/.build_release/examples/openpose/../../lib/libopenpose.so.1.0.1(_ZN2op14putTextOnCvMatERN2cv3MatERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEERKNS_5PointIiEERKNS0_7Scalar_IdEEb+0x95)[0x7f3c62cf8d95]

```

**Comment by gineshidalgo99** [comment]
OpenCV 3.3 might not be supported by Caffe yet.

Please, try 2.4.X to see if it's that the problem.

**Comment by gineshidalgo99** [comment]
(Closed given the no answer)

**Comment by yurochang** [comment]
I don't want to override my OpenCV3, So I cmake OpenCV2.4.13  in '/usr/local/opencv2/'. But Idon't know how to fix this in the Openpose 1.0.0, I add "set(OpenCV_DIR "/usr/local/opencv2/share/OpenCV")' but it still seems like use the default OpenCV3?
@gineshidalgo99
